pkg/gas/models.go

```diff
@@ -152,6 +153,8 @@ type EvmEstimator interface {
 	// GetLegacyGas Calculates initial gas fee for non-EIP1559 transaction
 	// maxGasPriceWei parameter is the highest possible gas fee cap that the function will return
 	GetLegacyGas(ctx context.Context, calldata []byte, gasLimit uint64, maxGasPriceWei *assets.Wei, opts ...fees.Opt) (gasPrice *assets.Wei, chainSpecificGasLimit uint64, err error)
+	// GetMaxLegacyGas returns the maximum gas price allowed by the connectivity threshold to improve inclusion probability.
```

Comment:
Probably good to add `non-EIP1559` here, e.g.:

```suggestion
	// GetMaxLegacyGas returns the maximum non-EIP1559 gas price allowed by the connectivity threshold to improve inclusion probability.
```

Reply:
That would imply there is EIP-1559 and non-EIP-1559 gas price. Maximum gas price seems reasonable to me.

Reply:
My point is that it's hard to deduce the meaning of `legacy`. If we add `non-EIP-1559` then the reader knows that's what `legacy` means. 
Similar to what's already in [line 153](https://github.com/smartcontractkit/chainlink-evm/pull/310/changes/BASE..11a87cb8733462dc81c7f5bbc55fe70910d07f27#diff-a91dda1a0cd687f57aab587546afa5d2b388bb38a0e1523e913aba44bf2f5bfaR153).

Reply:
I'm ok with rephrasing, but non-EIP1559 gas price is not a valid statement. How about:
`GetMaxLegacyGas returns the maximum non-EIP1559 gas fee allowed by the connectivity threshold to improve inclusion probability.`
What that be clearer?

Reply:
Yes!